src/less/radio/base/radio.less

```diff
@@ -21,11 +23,13 @@ svg.radio__unchecked {
 }
 
 svg.radio__checked {
-    color: @radio-checked-color;
+    // color: @radio-checked-color;
```

Comment:
PR looks good, just remove these two comments, revert the other file I mention, and then I think we are good to go,.

Reply:
And finished!